node-data/src/events.rs

```diff
@@ -5,6 +5,7 @@
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
 mod blocks;
+pub mod contract;
```

Comment:
nitpick: I suggest we group pub modules away from non pub modules and either add `//!` doc in the module or triple doc `///` above the pub module.